app/controllers/people_controller.rb

```diff
@@ -68,7 +71,7 @@ def inactive
   end
 
   def index
-    @people = Person.active.select{|person| person.department&.name == 'CERT' || person.department&.name == 'Police'}
+    @people = Person.active.select{|person| person.department&.shortname == 'CERT' || person.department&.shortname == 'BAUX'}
```

Comment:
Those are better solutions, but I must confess I did not give this my full attention. I believe this all goes away with your next PR ?